prow/cmd/build/controller.go

```diff
@@ -57,6 +58,20 @@ const (
 	controllerName = "prow-build-crd"
 )
 
+type pjClient struct {
+	pjc       prowjobset.Interface
+	namespace string
+}
+
+func (c *pjClient) ReplaceProwJob(name string, pj prowjobv1.ProwJob) (prowjobv1.ProwJob, error) {
```

Comment:
The signature is the same with the plank client https://github.com/kubernetes/test-infra/blob/bbad344943041822cfe15d2840bc64da62164550/prow/plank/controller.go#L47. 

We can change it to a pointer but then it will require an adapter also in plank.  Right now we can use directly that client in the aborter.